travis-scripts/ma-read.golden

```diff
@@ -0,0 +1,22 @@
+Mender artifact:
```

Comment:
Added golden files for verifying the output of the mender-artifact commands. These need to be updated given output changes ofc.